dbms/src/Flash/tests/gtest_join_executor.cpp

```diff
@@ -136,7 +136,13 @@ try
                                .build(context);
 
             {
+                context.context.setSetting("max_bytes_before_external_join", Field(static_cast<UInt64>(0)));
                 executeAndAssertColumnsEqual(request, expected_cols[i * simple_test_num + j]);
+
+                // for spill to disk tests
+                context.context.setSetting("max_bytes_before_external_join", Field(static_cast<UInt64>(10000)));
+                ASSERT_THROW(executeAndAssertColumnsEqual(request, expected_cols[i * simple_test_num + j], {1}), Exception);
+                executeAndAssertColumnsEqual(request, expected_cols[i * simple_test_num + j], {2, 5, 10});
```

Comment:
Use `executeStreams` directly?
Because `executeAndAssertColumnsEqual` will run query at least three times: disable planner, enable planner, enable pipeline.

Reply:
and then leave `executeAndAssertColumnsEqual ` as it is.

Reply:
done.